data/stories/metadata.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "title": {
+            "cs": "O perníkové chaloupce",
+            "uk": "Прянична хатинка"
+        },
+        "slug": "pernikova-chaloupka",
```

Comment:
přidal bych URL cestu na thumbnail + MP3 soubor (takto je umožníme pak třeba přesun CDN od json konfigurace apod).

Reply:
Dává smysl, jen budu mít nejspíš trochu problém vyřešit potřebné formáty skrz platformy web/iOS/Android.

Reply:
Web to asi bude mít jednoduší - SVG a url na CDN.

Ostatní budeme muset nasekat (můžeme udělat cli tool ze skriptů co už máme) a pak je otázka jak bude distribuce (jestli to zapečeme do aplikace tak pak stačí mít název pernikova-chakoupka@1x apod (jak je to nyní slovníku)

Reply:
Zrovna tady máme obrázky ručně kreslené a oscanované, takže bitmapy, konkrétně JPG. Ten se nedá univerzálně použít? 

Přiklad 
![](https://github.com/cesko-digital/movapp/blob/main/public/kids/cervena-karkulka.jpg)

Reply:
Záleží jak to bude velký. ale obecně to chceš nastříhat na různé velikosti - i na webu nechceš načítat obrázek větší než monitor dokáže zobrazit, nebo opačně, chceš načíst vyšší rozlišení protože monitor máš třeba 4k.

Reply:
Na webe by to za nas mal automaticky riešiť Vercel: https://vercel.com/docs/concepts/image-optimization
Ale pre aplikacie to asi treba spravit manualne